lib/src/boilerplate_suggestors/boilerplate_utilities.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:analyzer/dart/ast/ast.dart';
+
+class SemverHelper {
+  static Map _exportList;
+
+  void fromReport(String path) async {
+    var fileContents;
+    await File(path).readAsString().then((String contents) {
+      fileContents = contents;
+    });
+
+    Map decoded = jsonDecode(fileContents);
+    _exportList = decoded['exports'];
+  }
+
+  List<dynamic> getPublicExportLocations(ClassDeclaration node) {
+    final className = node.name.name;
+    String classKey;
+
+    _exportList.forEach((key, value) {
+      if (value['type'] == 'class' && value['grammar']['name'] == className) {
+        classKey = key;
+      }
+    });
+
+    return [classKey, _exportList[classKey]];
```

Comment:
Is returning the `classKey` necessary? If not, this API would be easier to use if it was just `_exportList[classKey]` that was returned (with specific typing).

Reply:
I added `classKey` to the return just because I wasn't sure what kind of information would be needed in the helpful messages and it seemed like something like `lib/web_skin_dart.dart/ButtonProps` might be useful. But I guess they would already know what file they're in so it's probably not necessary.

Reply:
That makes sense! If we were to do this, using a class to represent this would be preferred so that it's clear what the data being returned represents, and also so that we'd have better typing:
```dart
class ExportLocations {
  final String classKey;
  final List<String> exportList;

  ExportLocations(this.classKey, this.exportList);
}
```

But yeah, for this case we probably just need the list.